app/workers/send_base.rb

```diff
@@ -0,0 +1,28 @@
+module Workers
+  class SendBase < Base
+    sidekiq_options queue: :http, retry: 0
+
+    MAX_RETRIES = (rt = AppConfig.environment.sidekiq.retry.get) ? rt.to_i : 0
```

Comment:
`to_i` [returns 0 if an invalid string/nil is provided](http://ruby-doc.org/core-2.2.0/String.html#method-i-to_i):

```ruby
MAX_RETRIES = AppConfig.environment.sidekiq.retry.get.to_i
```